build.sbt

```
@@ -3,39 +3,35 @@
 // See http://www.foundweekends.org/giter8/testing.html#Using+the+Giter8Plugin for more details.
 
 ThisBuild / githubWorkflowBuild := Seq(
-  WorkflowStep.Sbt(
-    List("g8Test"),
-    name = Some("Test generated template")
-  ),
-  WorkflowStep.Run(
-    List(
-      "cd target/sbt-test/http4s-g8/scripted",
-      "sbt assembly",
-      "gu install native-image",
-      "cat native-image-readme.md | grep 'native-image  -H*' | sh"
-    ),
-    cond = Some("startsWith(matrix.java, 'graalvm-')"),
-    name = Some("Build native assembly")
-  )
+  WorkflowStep.Sbt(List("g8Test"), name = Some("Test generated template"))
 )
 
 val PrimaryOS = "ubuntu-latest"
 val MacOS = "macos-latest"
 ThisBuild / githubWorkflowOSes := Seq(PrimaryOS, MacOS)
+ThisBuild / githubWorkflowJavaVersions := Seq(
+  JavaSpec.temurin("8"),
+  JavaSpec.temurin("11"),
+  JavaSpec.temurin("17")
+)
```

If you care enough to bring it back

```suggestion
ThisBuild / githubWorkflowJavaVersions := Seq(
  JavaSpec.temurin("8"),
  JavaSpec.temurin("11"),
  JavaSpec.temurin("17"),
  JavaSpec.graalvm("11")
)
```

---

I'm not sure what's calling assembly then, since that got lost in the merge.

---

Oh, if there are additional CI steps necessary, I don't know about that, sorry!

---

I'll try to keep this as just the backport, but I won't try too hard.